### PR TITLE
Create Blog “hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications”

### PR DESCRIPTION
**Pull request by sseshachala**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+---
+<style>
+li {
+   font-size: 27px;
+   line-height: 33px;
+   max-width: none;
+}
+</style>
+
+In my first blog post on [HPE GreenLake for Private Cloud Enterprise (GLPCE): Exploring a flexible infrastructure resource pool](https://developer.hpe.com/blog/hpe-greenlake-for-private-cloud-enterprise--Exploring-a-flexible-infrastructure-resource-pool/)I underscored the flexible nature of HPE GreenLake for Private Cloud Enterprise infrastructure. Its transparent cost analytics stand out as a significant benefit, aiding organizations in making well-informed financial and infrastructural choices. In our present digital era, having the right infrastructure is imperative, a need that GLPCE aptly fulfills through its robust support for automated tools like Terraform and Ansible, ensuring seamless and efficient infrastructure management and scaling.
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:17]: Need a space before I underscored

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+In this second segment, I will delve further into the enhanced features of HPE GreenLake for Private Cloud Enterprise: Virtual Machines. This exploration will highlight the substantial support these services extend to modern businesses. By demystifying the intricacies of deploying and efficiently scaling traditional applications, GLPCE stands as a pivotal ally for contemporary enterprise operations.
+
+**Introduction**
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:20]: Use level 2 titles ## Introduction
Add carriage return after title.

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+The real advantage that HPE GreenLake for Private Cloud Enterprise delivers lies in its ability to deploy essential components like Tier 0 (top or external tier) and Tier 1 (middle or distribution tier) routers, load balancers, and firewalls. This ability enables customers to seamlessly establish a three-tier application in the private cloud environment, ensuring robustness and efficiency in their operations. Beyond this, HPE GreenLake for Private Cloud Enterprise empowers customers to construct a comprehensive blueprint of a three-tier application, leveraging it for consistent and repeatable deployments. This approach not only simplifies deployment challenges but also enhances the efficiency and reliability of deploying traditional three-tier applications in a cloud environment.
+
+![Figure 1: A Traditional n-tier application to be hosted on Virtual Machine Service](/img/a-traditional-n-tier-application-to-be-hosted-on-virtual-machine-service.png "Figure 1: A Traditional n-tier application to be hosted on Virtual Machine Service")
+
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:33]: Add title for figure 1 using this as in part 1.

`> > <span style="color:grey; font-family:Arial; font-size:1em">Screenshot TBD: TBD </span>`

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+Navigating the integration of these disparate systems requires a thoughtful approach, balancing the stability and reliability of traditional architectures with the innovative features of modern application development and deployment strategies. This blending ensures organizations can harness the full potential of both technological paradigms to drive operational excellence and competitive advantage in the marketplace. Addressing this challenge, the HPE GreenLake for Private Cloud Enterprise: Virtual Machines   serves as a comprehensive platform. It is designed to support the simultaneous operation and integration of both traditional and modern application types.
+
+**Traditional architecture**
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:25]: Make title 2

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+![Figure 1: A Traditional n-tier application to be hosted on Virtual Machine Service](/img/a-traditional-n-tier-application-to-be-hosted-on-virtual-machine-service.png "Figure 1: A Traditional n-tier application to be hosted on Virtual Machine Service")
+
+By integrating these capabilities, HPE GreenLake for Private Cloud Enterprise stands out as a solution that not only supports the deployment of traditional applications but also underscores the advancements in modern applications, ensuring seamless cloud transitions and robust integration. It adeptly positions enterprises to navigate the intricate waters of technological evolution, ensuring they remain at the forefront of innovation and operational efficiency.
+
+**Deploying and scaling traditional applications**
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:36]: Title and new line

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+   During the Runtime phase, instances are activated and monitored as they launch upon reaching scaling thresholds. 
+   For a clearer understanding, this post will include step-by-step screenshots. To start, log into the HPE GreenLake for Private Cloud Enterprise console and navigate to the Virtual Machines section on the dashboard.
+
+**Design and development phase**: In this section, you’ll learn how to manage groups and set scaling thresholds for CPU and memory.
+
+1. Creating a Group: Start by accessing the Virtual Machines link on the dashboard. Click on **Launch Service console** and then select **Groups** from the **Infrastructures** dropdown. Here, initiates the creation of a designated group, as illustrated in screenshot 1. This approach allows you to bundle associated resources, ensuring streamlined management and clear oversight.
+
+   ![Screenshot 1: Managing groups and setting scaled thresholds for CPU and memory.](/img/managing-groups-and-setting-scaled-thresholds-for-cpu-and-memory.png "Screenshot 1: Managing groups and setting scaled thresholds for CPU and memory.")
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:52]: Picture title

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+3. Create NSX-T Load Balancer: Screenshot 2 shows creating the **Scale-Post-LB** load balancer, set to **Small** size and **Enabled**, is in an Up administrative state. Connected to the main network router, **Tier-1 Gateway**, it logs activity at a Warning level. 
+      ![Screenshot 2: NSX-T Load Balancer](/img/nsx-t-load-balancer.png "Screenshot 2: NSX-T Load Balancer")
+   This tool distributes traffic across servers, enhancing performance and reliability, while granting access to all groups.
+4. Instance provisioning: Screenshot 3 takes you through the detailed steps of setting up an instance:
+
+a. Initiating instance creation: Begin by navigating to **Provisioning**, then select **Instances** and click on Create Instance. At this juncture, identify and assign the right group for your resources. It is now also the time to detail key attributes, such as the instance name, its operating environment, and any relevant labels. This meticulous labeling aids in precise resource tracking and management.
+
+![Screenshot 3: Creating and configuring instance](/img/screenshot-3-creating-and-configuring-instance.png "Screenshot 3: Creating and configuring instance")
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:61]: Picture title

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+b. Layout configuration: Subsequently, shift your focus to integrating the instance with a load balancer. While many prominent load-balancing options are available, including the likes of F5, the NSX-T Load Balancer (screenshot 2) native to HPE GreenLake for Private Cloud Enterprise  is a good fit. It's equipped with a comprehensive set of features tailored to meet our deployment requirements, ensuring even distribution of incoming traffic across multiple endpoints.
+c. Final settings: Concluding the process, determine the root volume size to match your data storage necessities. Concurrently, select and designate the network on which these instances will function. This guarantees flawless interaction and operation within the stipulated digital ecosystem.
+
+![Screenshot 4: Configuring automation, scale factor and load balancer](/img/configuring-automation-scale-factor-and-load-balancer.png "Screenshot 4: Configuring automation, scale factor and load balancer")
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:66]: picture Title

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+2. The Scale type selection follows, referencing the threshold previously established in Screenshot 1. For clarity, you can revisit Screenshot 1.
+3. In Screenshot 2, you’ll see how to set up an NSX-T Load Balancer. Port 80 is primarily designated for HTTP traffic. While port 80 is the focus, other ports like 443 for HTTPS or custom ports for database tasks (e.g., 1433, 1521) can be configured. It's crucial to note that, initially, configurations apply only to the servers behind the load balancer. When a new instance arises due to scaling thresholds, it inherits these settings but isn't automatically added to the load balancer. A separate script is required to include the new instance to the load balancer.
+4. It's essential to highlight that activating port 443 for HTTPS provides the capability to configure and implement SSL certificates, bolstering security.
+5. Additionally, there's flexibility to choose the installation of custom agents on the instances, such as third-party monitoring tools or security vulnerability scanners.
+
+   ![Screenshot 5: Tagging the resources.](/img/tagging-the-resources..png "Screenshot 5: Tagging the resources.")
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:76]: Picture title

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+
+Next, we will explore the two key phases of application deployment and scaling: 
+
+1. Design and Development
+2. Runtime
+   In the Design and Development phase, you will initialize hosting in the HPE GreenLake for Private Cloud Enterprise console and set up web layer instances using the c2i type. Utilizing Ansible, you will implement threshold-based scaling and integrate a load balancer to ensure continuous service. Although the primary focus is on port 80, other ports are also available.
+   During the Runtime phase, instances are activated and monitored as they launch upon reaching scaling thresholds. 
+   For a clearer understanding, this post will include step-by-step screenshots. To start, log into the HPE GreenLake for Private Cloud Enterprise console and navigate to the Virtual Machines section on the dashboard.
+
+**Design and development phase**: In this section, you’ll learn how to manage groups and set scaling thresholds for CPU and memory.
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:48]: Make title 2

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+4. It's essential to highlight that activating port 443 for HTTPS provides the capability to configure and implement SSL certificates, bolstering security.
+5. Additionally, there's flexibility to choose the installation of custom agents on the instances, such as third-party monitoring tools or security vulnerability scanners.
+
+   ![Screenshot 5: Tagging the resources.](/img/tagging-the-resources..png "Screenshot 5: Tagging the resources.")
+
+In Screenshot 5, you’ll notice an invaluable feature: the ability to create tags during the instance setup. Tagging resources allows organizations to effortlessly track and evaluate their resource consumption, providing insights into patterns, trends, and potential areas of optimization. Furthermore, these tags become indispensable when delving into spend analytics, helping to allocate and manage costs effectively. This functionality is especially crucial within the HPE GreenLake suite, as it promotes both transparency and strategic decision-making for businesses.
+
+**Runtime**: Observe screenshot 6, where you’ll notice instances are activated and are attached to load balancer.
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:80]: Make title

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+
+![Screenshot 6: Provisioning – 2 instances up and attached to load balancer.](/img/provisioning-–-2-instances-up-and-attached-to-load-balancer.png "Screenshot 6: Provisioning – 2 instances up and attached to load balancer.")
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:82]: Picture title

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+
+Provisioning: Screenshot 6 shows the final phase of the instance wizard: the Review screen. This overview permits users to review and affirm their established configurations. By clicking the complete button, the provisioning process is activated. Although the instances themselves do not undergo optimization, the earlier described workflow and automation contribute to enhancing application performance. This enhancement is achieved by efficiently scaling out or scaling in the number of virtual machines based on the predefined scale factor, ensuring optimal application operation without manually adjusting the VM count.
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:84]: Title 3 maybe?

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+
+![Screenshot 7: Scaled up VMs](/img/scaled-up-vms.png "Screenshot 7: Scaled up VMs")
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:86]: Picture title

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+
+Scaling observation: Now, draw your attention to the tripartite progression of instance scaling that is shown in Screenshot 7:
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:88]: make title

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 16]
```diff
@@ -0,0 +1,97 @@
...
+
+1. **Initial state**: This stage represents the baseline, illustrating the system's configuration before any significant memory demand. It's the foundation, where the system awaits triggers or conditions to initiate any scaling.
+2. **Intermediate phase**: During the transition, memory demand escalates and meets the scaling threshold. This juncture is pivotal — the system responds by initializing new instances. It's an adaptive phase, wherein the system's agility becomes evident, reacting in real-time to the increasing demands.
+3. F**inal state**: The culmination is a full display of responsiveness. With memory utilization hitting its designated peak, the system maximizes its resources. Four active instances, mirroring the set scaling threshold, now operate cohesively. This not only reflects an adaptive system but also showcases a setup geared for efficiency and robust performance, ensuring user demands are consistently met.
+
+**Conclusion**
```
Comment (Didier-Lalli) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:94]: Make title 2

**Review by Didier-Lalli — CHANGES_REQUESTED** [review]

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md (hunk 16 above)
Comment (Didier-Lalli) [reply content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:94]: Also in Final state the F is not bold.

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 17]
```diff
@@ -0,0 +1,97 @@
+---
+title: HPE GreenLake for Private Cloud Enterprise (GLPCE) Deploying and scaling
+  traditional applications
+date: 2023-10-03T03:45:24.053Z
+author: Sudheendra Seshachala Rao
+authorimage: /img/1695127848528.jpeg
+disable: false
+---
```
Comment (choukden) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:8]: Please add the tag:
hpe-greenlake-for-private-cloud-enterprise
to tie your blog post to the HPE GreenLake for private cloud enterprise landing page on the HPE Dev portal
https://developer.hpe.com/greenlake/hpe-greenlake-for-private-cloud-enterprise/home/

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md [hunk 18]
```diff
@@ -0,0 +1,97 @@
...
+title: HPE GreenLake for Private Cloud Enterprise (GLPCE) Deploying and scaling
```
Comment (choukden) [inline content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:2]: Please remove (GLCPE) from the title to be consistent with your first blog post of the series.

**Review by choukden — CHANGES_REQUESTED** [review]
Hi Sudheendra

I would really encourage you to apply the HPE GreenLake for Private cloud Enterprise branding convention as per my comments in the MS-Word docuemtn in your sharepoint.


I agree that for readers using GLCPE as abbreviation would probably be easier for reasing the post entry, however by experience, HPE is scrutinizing everything published, so sooner or later we will be asked to modify the blog post entry to compliant with the HPE Branding.

Thank you for taking into account this recommendation.

Best regards
Denis

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md (hunk 17 above)
Comment (sseshachala) [reply content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:11]: Done

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md (hunk 1 above)
Comment (sseshachala) [reply content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md]: Done

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md (hunk 2 above)
Comment (sseshachala) [reply content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:23]: Done

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md (hunk 3 above)
Comment (sseshachala) [reply content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:37]: Done

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md (hunk 4 above)
Comment (sseshachala) [reply content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:29]: done

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md (hunk 5 above)
Comment (sseshachala) [reply content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:43]: done

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md (hunk 12 above)
Comment (sseshachala) [reply content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:102]: done

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md (hunk 13 above)
Comment (sseshachala) [reply content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:106]: done

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md (hunk 14 above)
Comment (sseshachala) [reply content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:108]: done

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md (hunk 15 above)
Comment (sseshachala) [reply content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md:111]: done

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md (hunk 16 above)
Comment (sseshachala) [reply content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md]: done

File: content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md (hunk 18 above)
Comment (sseshachala) [reply content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md]: done

**Pull request merged**